Classes/CalcsTab.lua

```diff
@@ -487,9 +492,13 @@ function CalcsTabClass:PowerBuilder()
 								(output.Evasion - calcBase.Evasion) / m_max(10000, calcBase.Evasion) +
 								(output.LifeRegen - calcBase.LifeRegen) / 500 +
 								(output.EnergyShieldRegen - calcBase.EnergyShieldRegen) / 1000
-				if node.path then
+				if node.path or node.recipe and not node.ascendancyName then
 					newPowerMax.offence = m_max(newPowerMax.offence, node.power.offence)
 					newPowerMax.defence = m_max(newPowerMax.defence, node.power.defence)
+					local pathCost = node.path and #node.path or 1
```

Comment:
#1226 

Should be a fairly simple fix, but this line is causing anoint only nodes to be the highest priority nodes to suggest